From eca38387818779a0bd67477295f785749de94f4c Mon Sep 17 00:00:00 2001
From: Robin Gong <yibin.gong@nxp.com>
Date: Fri, 29 Apr 2016 15:46:30 +0800
Subject: [PATCH 1196/5242] MLK-12928-13 ARM: dts: imx6ul-14x14-evk-pf1550:
 add onkey device node

commit  890e6a154dd3035321b50a7d736c2a17f3a2503d from
https://source.codeaurora.org/external/imx/linux-imx.git

Add onkey device node in dts.

Signed-off-by: Robin Gong <yibin.gong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ul-14x14-evk-pf1550.dts |    8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6ul-14x14-evk-pf1550.dts b/arch/arm/boot/dts/imx6ul-14x14-evk-pf1550.dts
index 5c60b1a..9e48a0d 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-evk-pf1550.dts
+++ b/arch/arm/boot/dts/imx6ul-14x14-evk-pf1550.dts
@@ -46,8 +46,14 @@
 		reg = <0x08>;
 		pinctrl-0 = <&pinctrl_pf1550>;
 
+		onkey {
+			compatible = "fsl,pf1550-onkey";
+			linux,keycode = <KEY_POWER>;
+			wakeup;
+		};
+
 		regulators {
-			compatible = "pf1550-regulator";
+			compatible = "fsl,pf1550-regulator";
 
 			sw1_reg: SW1 {
 				regulator-name = "SW1";
-- 
1.7.9.5

